packages/analyzer/src/javascript/esutil.ts

```diff
@@ -104,6 +104,18 @@ export function* getSimpleObjectProperties(node: babel.ObjectExpression) {
   }
 }
 
+/** Like getSimpleObjectProperties but deals with paths. */
```

Comment:
Can you give me an example to help me understand what a "path" means here?

Reply:
A `NodePath` is a richer API wrapped around a babel ast node